Remove duplicate setup instructions from android custom build page (#4869)
godotengine/godot-docs: tutorials/export/android_custom_build.rst

* Modify the project before it's built.
* Add external SDKs that build with your project.

> Configuring the custom build is a more or less straightforward process. But first
> you need to do the steps in :ref:`exporting for android<doc_exporting_for_android>`
> up to "Setting it up in Godot." After doing that follow the steps below.

Reviewer: ```suggestion
Configuring the custom build is a fairly straightforward process. But first,
you need to follow the steps in :ref:`exporting for android<doc_exporting_for_android>`
up to **Setting it up in Godot**. After doing that, follow the steps below.
```
